.ci/scripts/ui.sh

```diff
@@ -3,4 +3,4 @@
 
 cd scripts/kibana/validate-ts-interfaces-against-apm-server-sample-docs
 yarn
-yarn setup && yarn lint
+yarn setup elastic "${INTEGRATION_TESTING_VERSION}" elastic "${INTEGRATION_TESTING_VERSION}" && yarn lint
```

Comment:
Is the double `elastic "${INTEGRATION_TESTING_VERSION}"` block correct?

Reply:
yes, it is correct https://github.com/elastic/apm-integration-testing/blob/master/scripts/kibana/validate-ts-interfaces-against-apm-server-sample-docs/scripts/setup.sh#L6-L9